source/standards/configuration-management.html.md.erb

```diff
@@ -0,0 +1,25 @@
+---
+title: Use configuration management
+expires: 2018-05-16
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+You should use configuration management to track and control changes in large software deployments. When using configuration management you ensure your environment configuration is documented and [stored as code][].
+
+For example, when you run virtual machines (VM) in [Amazon Elastic Compute Cloud (Amazon EC2)][] and the VMs instance configuration becomes too complex to manage either during its deployment or within your [Amazon Machine Image (AMI)][] use configuration management.
+
+You should use [Puppet][] in large software environments, and for smaller environments, you can use [Ansible][].
```

Comment:
I'd echo Chris's comment, we could with an example of what large and small means in this context.